src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPageFairRecoveryOasis.java

```diff
@@ -19,15 +17,23 @@
 
   private static Dimension DESKTOP_SIZE = new Dimension(1920, 768);
 
+  private static final String WIKIA = "arecovery";
+  private static final String WIKIA_ARTICLE = "SyntheticTests/Static_image";
+  private static final String INSTART_LOGIC_SWITCHED_OFF = "InstantGlobals.wgAdDriverInstartLogicRecoveryCountries=[ZZ]";
+  private static final String INSTANT_GLOBAL_PREMIUM_AD_LAYOUT_SWITCHED_OFF = "InstantGlobals.wgAdDriverPremiumAdLayoutCountries=[ZZ]";
+  private static final String INSTANT_GLOBAL_PAGE_FAIR_SWITCHED_ON = "InstantGlobals.wgAdDriverPageFairRecoveryCountries=[XX]";
+
+  private static final String RECOVERY_ARTICLE = WIKIA_ARTICLE + "?" + INSTANT_GLOBAL_PREMIUM_AD_LAYOUT_SWITCHED_OFF +
```

Comment:
Consider building this in a new method (which you re-use in the test methods) and using `format()` in order to make it more readable.